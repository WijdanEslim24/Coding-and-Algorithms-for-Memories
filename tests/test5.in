6
slam
almi
lino
mino
smio
alai
4