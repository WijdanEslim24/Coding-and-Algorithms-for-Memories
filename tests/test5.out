SCS: aslamino
SCS Length: 8
